react-big-calendar/index.d.ts

```diff
@@ -48,5 +50,9 @@ declare module 'react-big-calendar' {
         static globalizeLocalizer(globalizeInstance: Object): void;
     }
 
-     export default BigCalendar;
+    /* This enables 'import * as BigCalendar' syntax when compiling to es2015 */
+    namespace BigCalendar {}
+
+    /* react-big-calendar is exported as a commonjs module (it uses babel-preset-jason) */
+    export = BigCalendar;
```

Comment:
You are hacking incorrect behavior, please check:
- https://stackoverflow.com/questions/39415661/why-cant-i-import-a-class-or-function-with-import-as-x-from-y  
- https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes

From documentation: http://intljusticemission.github.io/react-big-calendar/examples/index.html#intro
```
import BigCalendar from 'react-big-calendar';
```
This documentation implies a Class with static methods and not a namespace.
Previous declaration was 100% correct and correctly guarded for right usage, you shouldn't be able to import by 'import * as BigCalendar, because this would lead to errors in TS projects.